sdlib/d/gc/tcache.d

```diff
@@ -727,6 +727,20 @@ private:
 		return true;
 	}
 
+	bool growLarge(PageDescriptor pd, ref CachedExtentMap emap, uint npages) {
```

Comment:
You don't need to pass the emap down, you have it in the thread cache.

Reply:
um... oh yeah. Fixed.